### PR TITLE
 Friendly approach when item not found

### PR DESCRIPTION
**Pull request by mbdwey**

### CODE REVIEW

File: app/Http/Controllers/Settings/Settings.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public function update(Request $request)
         $fields = $request->all();
         $company_id = $request->get('company_id');
 
-        if (empty($company_id)) {
+        if (empty($company_id) || !env('APP_INSTALLED')) {
```
[the remark this answers is not in the document]
Comment (mbdwey) [reply app/Http/Controllers/Settings/Settings.php]: No, Sorry will remove it

File: resources/lang/en-GB/general.php [hunk 2]
```diff
@@ -111,5 +111,6 @@
         ],
         'no_file_selected'  => 'No file selected...',
     ],
-
+  'Item not found' => 'Item not found',
```
[the remark this answers is not in the document]
Comment (mbdwey) [reply resources/lang/en-GB/general.php]: I can't understand your request !

File: resources/views/common/404-item.blade.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
+@extends('layouts.modules')
+
+@section('title', __('general.dashboard'))
```
[the remark this answers is not in the document]
Comment (mbdwey) [reply resources/views/common/404-item.blade.php]: Sure

File: resources/lang/en-GB/general.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (denisdulici) [reply resources/lang/en-GB/general.php]: Use `item_not_found` instead of `Item not found` as key. Check out the article I referred for more details.

**Pull request closed without merging**